sdk/iothub/azure-mgmt-iotcentral/CHANGELOG.md

```diff
@@ -1,26 +1,90 @@
 # Release History
 
-## 5.0.0 (2021-05-13)
+## 5.0.0b1 (2021-05-13)
```

Comment:
(1)the latest track1 version for 'iotcentral' is 4.1.0(https://pypi.org/project/azure-mgmt-iotcentral/#history). The first track2 version should be 5 larger than that. So the version should be 9.0.0b1. 
(2)copy the lastest CHANGELOG from 'release/v3', then add '9.0.0b1' changelog content.